views/wildlife_views.py

```diff
@@ -53,38 +53,88 @@ def get_single_wildlife_type(id):
 
 
 def get_wildlife_by_park_id(park_id):
-    """A function that takes in one parameter and used to retrieve information about wildlife in a specific park."""
-    with sqlite3.connect("./national_park.sqlite3") as conn:
-        conn.row_factory = sqlite3.Row
-        db_cursor = conn.cursor()
+    """getting all wildlife in a specific park"""
 
-        db_cursor.execute("""
+    sql = """
         SELECT
-        w.id,
-        w.name,
-		w.information,
-		w.wildlife_group_id,
-		w.image
+            w.id,
+            w.name,
+            w.information,
+            w.wildlife_group_id,
+            w.image
         FROM Park_Wildlife pw
         JOIN Wildlife w
             ON pw.wildlife_id = w.id
         WHERE pw.park_id = ?
 		ORDER BY w.name ASC;
-        """, (park_id, ))
+    """
+
+    all_wildlife = []
+
+    dataset = get_all_by_param(sql, park_id)
+    if not dataset:
```

Comment:
This is non-Pythonic _(i.e. checking for falsy values like in JavaScript)_. To be Pythonic check for exact value.

```py
if dataset is None:
```